.ci/travis-script.sh

```diff
@@ -2,13 +2,21 @@
 
 set -ev
 
-BASE_CMD="docker-compose -f .travis.compose.yml run --rm test-hbmpc"
```

Comment:
Not sure if we need to actually rm the docker container, but if we don't `--rm` it right away, then we can copy over the coverage report for use by `codecov`